test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -1487,7 +1487,7 @@ public void PackCommand_PackProject_PacksFromNuspec_InstallPackageToOutputPath()
             }
         }
 
-        [PlatformTheory(Platform.Windows, Skip = "https://github.com/dotnet/sdk/issues/28131")]
+        [PlatformTheory(Platform.Windows)]
```

Comment:
Can we run this test across all platforms instead of only on Windows?

Reply:
Possibly, but outside the scope of this PR.  I'm not sure why it was Windows-only to begin with.

I created a [separate issue](https://github.com/NuGet/Home/issues/12364) tracking that.